### PR TITLE
Update the cel-go workspace to point to latest cel-spec

### PR DESCRIPTION
**Pull request by TristonianJones**
Updates the workspace to refer to latest changes present in the cel-spec
proto definitions. Namely replaces `Constant` with `Literal`

### CODE REVIEW

File: checker/checker.go [hunk 1]
```diff
@@ -73,23 +73,23 @@ func (c *checker) check(e *expr.Expr) {
...
-		case *expr.Constant_Int64Value:
-			c.checkInt64Constant(e)
-		case *expr.Constant_NullValue:
-			c.checkNullConstant(e)
-		case *expr.Constant_StringValue:
-			c.checkStringConstant(e)
-		case *expr.Constant_Uint64Value:
-			c.checkUint64Constant(e)
+	case *expr.Expr_LiteralExpr:
+		constExpr := e.GetLiteralExpr()
```
Comment (JimLarson) [inline checker/checker.go]: Rename "constExpr" to "literalExpr" or just "literal".

File: interpreter/astwalker.go [hunk 2]
```diff
@@ -119,27 +119,23 @@ func (w *astWalker) walk(node *expr.Expr) []Instruction {
 }
 
 func (w *astWalker) walkConst(node *expr.Expr) Instruction {
```
Comment (JimLarson) [inline interpreter/astwalker.go]: "walkConst" -> "walkLiteral" ?

File: interpreter/astwalker.go [hunk 3]
```diff
@@ -119,27 +119,23 @@ func (w *astWalker) walk(node *expr.Expr) []Instruction {
...
+	case *expr.Literal_Int64Value:
+		value = types.Int(literalExpr.GetInt64Value())
+	case *expr.Literal_NullValue:
+		value = types.Null(literalExpr.GetNullValue())
+	case *expr.Literal_StringValue:
+		value = types.String(literalExpr.GetStringValue())
+	case *expr.Literal_Uint64Value:
+		value = types.Uint(literalExpr.GetUint64Value())
 	}
 	return NewConst(node.Id, value)
```
Comment (JimLarson) [inline interpreter/astwalker.go]: NewConst -> NewLiteral ?  I can't see the context to tell whether this is appropriate.

File: interpreter/astwalker.go [hunk 4]
```diff
@@ -223,17 +219,17 @@ func (w *astWalker) walkCall(node *expr.Expr) []Instruction {
 		instructions = append(instructions, condition...)
 		// 1: jump to <END> on undefined/error
 		instructions = append(instructions,
-			NewJump(conditionId, len(trueVal)+len(falseVal) + 3,
+			NewJump(conditionId, len(trueVal)+len(falseVal)+3,
```
Comment (JimLarson) [inline interpreter/astwalker.go:222]: Formatting is backsliding?  Is this the way gofmt wants it?

File: parser/helper.go [hunk 5]
```diff
@@ -51,34 +51,34 @@ func (p *parserHelper) reportError(ctx interface{}, format string, args ...inter
...
 
-func (p *parserHelper) newConstant(ctx interface{}, value *expr.Constant) *expr.Expr {
+func (p *parserHelper) newLiteral(ctx interface{}, value *expr.Literal) *expr.Expr {
 	exprNode := p.newExpr(ctx)
-	exprNode.ExprKind = &expr.Expr_ConstExpr{ConstExpr: value}
+	exprNode.ExprKind = &expr.Expr_LiteralExpr{LiteralExpr: value}
 	return exprNode
 }
 
 func (p *parserHelper) newConstBool(ctx interface{}, value bool) *expr.Expr {
```
Comment (JimLarson) [inline parser/helper.go]: newConstBool -> newLiteralBool ?  And similar below...

File: test/expr.go [hunk 6]
```diff
@@ -175,38 +175,32 @@ func ExprSelect(id int64, operand *expr.Expr, field string) *expr.Expr {
 }
 
 func ExprConst(id int64, value interface{}) *expr.Expr {
```
Comment (JimLarson) [inline test/expr.go]: ExprConst -> ExprLiteral ?

File: interpreter/astwalker.go (hunk 4 above)
Comment (TristonianJones) [reply interpreter/astwalker.go:222]: Hmm, this must be what go fmt wants.

**Review by JimLarson — APPROVED** [review]
Love it.

**Pull request merged**